conf/messages

```diff
@@ -274,11 +274,21 @@ ssttp.you-need-to-file.other-tax=If you’re unsure how to proceed for any reaso
 ssttp.you-need-to-file.other-tax.number=0300 200 3504
 
 ssttp.arrangement.instalment-summary.title=Your instalment plan summary
+<<<<<<< HEAD
```

Comment:
The merge has left in the conflict lines and as such has stopped the branch from compiling so these will need removing as well adding in loads of duplicated messages